test/chrono-test.cc

```diff
@@ -533,6 +537,8 @@ TEST(chrono_test, format_specs) {
   EXPECT_EQ("12", fmt::format("{:%I}", std::chrono::hours(24)));
   EXPECT_EQ("04", fmt::format("{:%I}", std::chrono::hours(4)));
   EXPECT_EQ("02", fmt::format("{:%I}", std::chrono::hours(14)));
+  EXPECT_EQ("12345", fmt::format("{:%j}", days(12345)));
+  EXPECT_EQ("12345", fmt::format("{:%j}", std::chrono::hours(12345*24+12)));
```

Comment:
nit: spaces around operators (*, +) or apply clang-format

Reply:
Sorry, fixed.

(clang-format gave a few other changes, I did not apply those changes would be unrelated to this PR)